sensors/mediation/sensor_hal_default.te

```diff
@@ -2,9 +2,12 @@ allow hal_sensors_default self:socket create_socket_perms;
 allowxperm hal_sensors_default self:socket ioctl unpriv_sock_ioctls;
 allow hal_sensors_default serial_device:chr_file rw_file_perms;
 
-userdebug_or_eng(`
-    permissive hal_sensors_default;
-    dontaudit hal_sensors_default default_prop:file { open read getattr map };
-    dontaudit hal_sensors_default port:tcp_socket { name_connect };
-    dontaudit hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
-')
+allow hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
+dontaudit hal_sensors_default default_prop:file { open read getattr map };
+allow hal_sensors_default port:tcp_socket { name_connect };
+
+#=================hal sensors_default=================
+allow hal_sensors_default vendor_intel_ipaddr_prop:file read;
```

Comment:
use get_prop(hal_sensors_default, vendor_intel_ipaddr_prop) instead of line10 ~ line13